platform-coop-toolkit/coop-map-directory-index
# feat: update Organization.geo_scope to match data (fix #64)

## Description

This PR brings the model for `Organization.geo_scope` in line with what's in the database.

## Steps to test

1. Check out this branch.
2. Run the migration.
3. Try to edit an organization in the admin, setting/modifying values for _Geographic scope_.

**Expected behavior:** All geographic scope values can be saved to the database.

## Additional information

Not applicable.

## Related issues

Depends on #91.
Resolves #64.
